src/EluvioLive.js

```diff
@@ -1180,62 +1295,64 @@ Lookup NFT: https://wallet.contentfabric.io/lookup/`; */
       resolveLinks: false,
     });
 
-    var pnft = await this.NftMake({ assetMetadata: m, hash });
-    console.log(pnft);
+    var pnft;
+    var pnfts = [];
 
-    var e = await this.client.EditContentObject({
-      libraryId,
-      objectId,
-    });
+    // Determine if this is a single or multi-image NFT
 
-    await this.client.ReplaceMetadata({
-      libraryId,
-      objectId,
-      writeToken: e.write_token,
-      metadataSubtree: "public/nft",
-      metadata: pnft,
-    });
+    if (imageDir != null && imageDir.length > 0) {
```

Comment:
this might not work since imageDir == undefined when not passed in? We might want to explicitly set a default param or just `if (imageDir && imageDir.length > 0)`

Reply:
Ok I changed to just imageDir && imageDir.length > 0